bundle/src/main/java/com/adobe/acs/commons/redirectmaps/package-info.java

```diff
@@ -17,5 +17,5 @@
  * limitations under the License.
  * #L%
  */
-@aQute.bnd.annotation.Version("3.1.0")
+@aQute.bnd.annotation.Version("3.14.4")
```

Comment:
I don't think this change is necessary. I only set the version to 3.14.4 on the models package because it wasn't set before and thus was getting the bundle version by default so I was just locking it in as 3.14.4. In this case, we already set a version, so there's no need to change.

Reply:
Sure @justinedelson -- I thought I saw baseline complaining previously, but it worked fine this time so rolling it back to 3.1.0